frontend/lib/qr_code_scanner/qr_overlay_shape.dart

```diff
@@ -0,0 +1,178 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+// temporary solution
+// checkout https://github.com/juliansteenbakker/mobile_scanner/issues/250#issuecomment-1209723708
```

Comment:
This issue mentions that a solution is being worked on which is already on their main: https://github.com/juliansteenbakker/mobile_scanner/blob/master/example/lib/barcode_scanner_window.dart

Reply:
We could temporarlily depend on their Github version instead of the published release if that simplifies this file.

Reply:
If I understand correctly, this is just a visual overlay to allow scanning qr codes inside of the overlay? We could just ignore that for now, until the feature has officially landed in mobile_scanner. (So we could just remove this overlay shape and the `_calculateScanArea` function in `qr_code_scanner_controls.dart`)

However, if it works without problems like this, we can also leave it like this I think :) (And maybe refactor once the feature has landed)